README-National-Cloud.md

```diff
@@ -0,0 +1,193 @@
+---
+services: active-directory
+platforms: dotnet
+author: negoe 
+level: 200
+client: .NET Web App (MVC)
+service: Microsoft Graph
+endpoint: AAD v1.0
+---
+
+
+# Build a multi-tenant SaaS web application using Azure AD & OpenID Connect
+
+[![Build status](https://identitydivision.visualstudio.com/IDDP/_apis/build/status/AAD%20Samples/.NET%20client%20samples/CI%20of%20Azure-Samples%20--%20active-directory-dotnet-webapp-multitenant-openidconnect)](https://identitydivision.visualstudio.com/IDDP/_build/latest?definitionId=729)
+
+## About this sample
+
+This sample shows how to build an ASP.NET MVC web application that uses [OpenID Connect](https://docs.microsoft.com/en-us/azure/active-directory/develop/v1-protocols-openid-connect-code) to sign up and sign in users from any Azure Active Directory tenant, using the [ASP.Net OpenID Connect OWIN middleware](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet) and the [Active Directory Authentication Library (ADAL) for .NET](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet). 
+
+It also introduces developers to the concept of 
+
+- Microsoft National cloud environments
+- Multi-tenant Azure Active Directory application
+
+
```

Comment:
You might want to add a small paragraph

>[!Note] If you want to run this sample in  **Microsoft worldwide Cloud**, navigate to the [README.md](./README-.md).

?
or something like that? to cross links both .md files?

Reply:
yes, that's absolutely make sense.

Reply:
+1

---
In reply to: [254300546](https://github.com/Azure-Samples/active-directory-dotnet-webapp-multitenant-openidconnect/pull/37#discussion_r254300546) [](ancestors = 254300546)